src/cloudConsole.ts

```diff
@@ -243,5 +243,5 @@ async function exec(command: string) {
 }
 
 function escapeFile(data: string): string {
-	return data.replace(/"/g, '\\"');
+	return data.replace(/\\/g, '\\\\\\\\').replace(/"/g, '\\"');
```

Comment:
replace with '\\\\\\' works for my testing. any special testing cases required extra escape in '\\\\\\\\' ?